src/ntfs/read.rs

```diff
@@ -106,82 +105,57 @@ pub fn manage_stops(collections: &mut Collections, path: &path::Path) -> Result<
     Ok(())
 }
 
-impl From<Price> for TicketV1 {
-    fn from(price: Price) -> Self {
-        TicketV1 {
-            id: price.id,
-            start_date: price.start_date,
-            end_date: price.end_date,
-            price: f64::from(price.price) / 100.,
-            currency_type: "EUR".into(),
-            validity_duration: None,
-            transfers: None,
-        }
-    }
-}
-
 pub fn manage_fares_v1(collections: &mut Collections, base_path: &path::Path) -> Result<()> {
-    let file = "prices.csv";
-    if !base_path.join(file).exists() {
-        info!("Skipping fares");
+    let file_prices = "prices.csv";
+    let file_od_fares = "od_fares.csv";
+    let file_fares = "fares.csv";
+
+    if !base_path.join(file_prices).exists() || !base_path.join(file_od_fares).exists() {
+        info!("Skipping {} and {}", file_prices, file_od_fares);
```

Comment:
I'm not sure this is the correct message.
you should skip all the reading of fares

Reply:
Ok I understand what you mean. All 3 files are skipped here (even if 1 is optional).